ApollosProject/apollos-prototype
# Fixes tab view cant scroll devos android

## DESCRIPTION

![2019-11-08 09 01 52](https://user-images.githubusercontent.com/1637694/68483219-1fd16800-0209-11ea-8f60-87fa61c24a59.gif)

^ Nuff said ;) 

### What does this PR do, or why is it needed?

### What design trade-offs/decisions were made?

All HorizontalTileFeeds are now wrapped with the GestureHandler enabled FlatList. This shouldn't have any side effects, except improving compatibility when using the HorizontalTileFeed with other GestureHandler components.

### How do I test this PR?

On both iOS and Android...
1. Open a devo.
2. Swipe to scripture, swipe back.
3. Swipe the sibling items 
4. Logout
5. "Login with Username/Password"
6. Make sure Login/Register tabview works.

### What automated tests did you write?

Updated snaps.

## TODO:

- [x] Closes #1157 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
